pkg/scheduler/framework/frameworkutils.go

```diff
@@ -203,6 +198,33 @@ func crossReferencePickedCustersAndObsoleteBindings(
 	return toCreate, toDelete, toPatch, nil
 }
 
+func patchBinding(binding *placementv1beta1.ClusterResourceBinding, scored *ScoredCluster, policy *placementv1beta1.ClusterSchedulingPolicySnapshot) *bindingWithPatch {
+	// Update the binding so that it is associated with the latest score.
+	updated := binding.DeepCopy()
+	affinityScore := int32(scored.Score.AffinityScore)
+	topologySpreadScore := int32(scored.Score.TopologySpreadScore)
+	// set the state as it might be unscheduled before
+	updated.Spec.State = placementv1beta1.BindingStateScheduled
```

Comment:
Hi Ryan! This is not necessarily the case; obsolete binding can also be in the Bound state.